scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
Hmmm ... looks like the settings infrastructure is [designed](https://github.com/CenterForOpenScience/osf.io/blob/21a0298131884ba3c1e31bb6e656572ecd130f00/website/settings/__init__.py) to use attributes of `website.settings` ... but that's not what we're doing here. We're importing directly from `website.settings.default` ... and yet I believe I'm seeing the `website.settings.local` values prevail(?!).